src/test/scripts/product_versions.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import urllib.request
+import json
+
+known_lts_version = {
```

Comment:
Instead of `known_lts_version`, will something like minimum supported version be better suited the purpose here?

Reply:
Good suggestion, I will change it.